### DISCUSSION TITLE
How to save the image with specified location and range on osgEarth Earth model to a file through code？

### DISCUSSION THREAD

**Discussion by Damengxin-123** [pelicanmapping/osgearth#2189] [Support Q&A] [answered]
I can confirm the range through coordinates, and then how to capture the elevation map, topographic map, vector map, etc. of the range on the earth model and store them in "png" orother files.

**Comment by gwaldron** [comment, marked as answer]
Every image layer has a `createImage` function. Figure out the tilekey you want by calling `Profile::createTileKey(x, y, level)` fora  point, or `Profile::getIntersectingTiles` for a region. Then you can call `createImage` for each tilekey to get the image. Good luck.
